react/features/base/config/functions.js

```diff
@@ -171,7 +189,11 @@ export function overrideConfigJSON(
             configObj = loggingConfig;
         }
         if (configObj) {
-            const configJSON = json[configName];
+            // whitelist only config.js, skip this for others
+            const configJSON
+                = configName === 'config'
```

Comment:
i thought @virtuacoplenny had found that the other config files had urls that could be problematic too.  lenny can confirm.

Reply:
I havn't seen such, but I can miss something :)

Reply:
The ones in interfaceConfig are not as bad as what was on the config blacklist because they require user interaction to be a problem. They're still potentially arbitrary URLs though, which is why interfaceConfig was mentioned.